layouts/partials/primary_bottom.html

```diff
@@ -3,8 +3,11 @@
 {{ $skipPageNav := .Scratch.Get "skipPageNav" }}
 
             {{ if .Scratch.Get "seeAlso" }}
-                {{ $related := .Site.RegularPages.Related . | first 6 }}
-                {{ with $related }}
+                {{ $related := .Site.RegularPages.Related . | first 8 }}
+                {{ $mixerSPOF := where $related "RelPermalink" "/blog/2017/mixer-spof-myth/"}}
+                {{ $mixerAdapter := where $related "RelPermalink" "/blog/2017/adapter-model/"}}
+                {{ $seeAlso := $related | complement $mixerSPOF $mixerAdapter | first 6}}
+                {{ with $seeAlso }}
```

Comment:
@davidhauck Does this sound OK?

Reply:
I'd rather have these pages be in a constant somewhere. That could be a list of directories in data/*.yaml or using [front-matter](https://gohugo.io/content-management/front-matter/#user-defined) on pages that we don't want appearing here, and then filtering on that instead. The latter is probably preferred.

Reply:
OK. switched to the front-matter approach. ptal.

Reply:
Sorry missed this, looks good!